crates/engine/tree/src/tree/mod.rs

```diff
@@ -169,7 +169,7 @@ impl TreeState {
         current_hash = self.canonical_block_hash();
         while let Some(current_block) = self.block_by_hash(current_hash) {
             if current_block.hash() == target_hash {
-                return false
+                return false;
```

Comment:
one nit: let's remove the added semicolons

Reply:
no problem